contrib/features/README.md

```diff
@@ -0,0 +1,7 @@
+## Design Documents
+
+This directory contains design information for Podman features that are being worked on or will be worked on in the future.
+All documents in this directory should be based on the [template](./TEMPLATE.md) provided.
+Major features should be preceded by design documents.
+Discussion on new features should take place on the pull request to add the design document.
+While we welcome feedback on existing design documents, anything that has already committed likely has work ongoing to implement it, and as such major design changes are unlikely to be possible.
```

Comment:
how should 'committed' be defined here?  I'm thinking merged? And speaking of which, whats the policy of 'this docis good"? Today, that's me as architect.  im thinking it needs to be core maintainers but not ALL of them.

Reply:
thought would be three core maintainers when authored by a non-core maintainer.  And two when it is.  probably should cross ref this in GOVERNANCE.md as well